py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -233,6 +233,39 @@ def aten_ops_embedding(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.embedding_bag.default)  # type: ignore[misc]
```

Comment:
For this, we'll need some sort of capability validator which ensures the third input is a numpy array or Torch tensor. The only way to have an input be a numpy array or Torch tensor is for it to be a `"get_attr"` node, since only constants can have their data accessed.

```python
def embedding_bag_validator(node: torch.fx.Node) -> bool:
    return node.args[2].op == "get_attr"
```